config/newrelic.yml

```diff
@@ -33,6 +33,7 @@ common: &default_settings
 development:
   <<: *default_settings
   app_name: Sara Alert (Development)
+  monitor_mode: <%= ENV['APP_IN_CI'].nil? ? true : false %>
```

Comment:
If we are in development I would say that this should just be `false` so that the benchmarks can be run locally without having to set `APP_IN_C`I environment variable or edit this setting.